msteams-platform/concepts/design/design-teams-app-basic-ui-components.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Design your app with basic Fluent UI components
 author: heath-hamilton
-description: Learn about the primitive UI components commonly used to build Microsoft Teams apps and get the Microsoft Teams UI Kit. Learn about components, such as breadcrumbs, card, carousel, dropdown, coachmark, picker, pivot, toggle, tooltip, and more. 
+description: Learn the UI components used to build Microsoft Teams apps and get the Teams UI Kit. Learn about fluent kit breadcrumb button card carousel dropdown coachmark picker pivot toggle tooltip
```

Comment:
@surbhigupta12 , this is keyword stuffing and not advised. Also, fluent kit casing is incorrect. Please get this addressed.